tests/metadata_with_image.py

```diff
@@ -18,6 +18,5 @@
 metadata = request.get_metadata()
 request.release()  # requests must always be returned to libcamera
 
-image.show()
```

Comment:
This blocks and pops a window up if there is an available viewport :face_exhaling:

Reply:
🔪